logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/CallRepository.kt

```diff
@@ -41,18 +42,25 @@ internal class CallDataSource(
         callApi.connectToSFT(url = url, data = data)
     }
 
+    override fun updateCallProfileFlow(callProfile: CallProfile) {
+        _callProfile.value = callProfile
+    }
+
     override fun getAllCalls(): StateFlow<List<Call>> = MutableStateFlow(allCalls.value.calls.values.toList())
 
     override fun getIncomingCalls(): Flow<List<Call>> = allCalls.map {
         it.calls.values.filter { call ->
-            call.status in listOf(
-                CallStatus.INCOMING
-            )
+            call.status == CallStatus.INCOMING
         }
     }
 
-    override fun getOngoingCall(): Flow<List<Call>> = allCalls.map {
-        it.calls.values.filter { call -> call.status == CallStatus.ESTABLISHED }
+        override fun getOngoingCall(): Flow<List<Call>> = allCalls.map {
```

Comment:
Since you're changing the signature, could we rename it to something like `ongoingCallsFlow` or `observeOngoingCalls` to better reflect what it returns?